src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/modalwindows/FacebookSignupModalComponentObject.java

```diff
@@ -117,4 +125,16 @@ public void createAccountNoEmail(String email, String emailPassword, String user
     typeEmail(email);
     createAccount();
   }
+
+  public void loginExistingAccount(String userName, String password) {
+    waitForElementByElement(existingUsernameField);
+    existingUsernameField.sendKeys(userName);
+    PageObjectLogging.log("loginExistingAccount", "username " + userName + " typed into the field", true);
+    waitForElementByElement(existingPasswordField);
+    existingPasswordField.sendKeys(password);
+    PageObjectLogging.log("loginExistingAccount", "password " + password + " typed into the field", true);
+    loginExistingButton.click();
+    PageObjectLogging.log("loginExistingAccount", "login button clicked", true);
```

Comment:
We have global event listener which log each click, so you don't need to do it by yourself.